pkgs/games/openra-mods/launch-game.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/sh
```

Comment:
Great wrapper. Probably doesn't belong here, though.

Reply:
Where else would it go?